### ISSUE TITLE
Align on how to display individual criteria on index of translations

### ISSUE THREAD

**Issue by clausmullie** [standard-for-public-code/community-translations-standard#68]
The index of the translations has a header for criteria, with all individual criteria nested underneath.

The da and es versions have a manually typed list.

The zh_Hant_TW version uses liquid. 

The standard repo does it with liquid as well. 

Should this align to all using liquid? This seems better than manually updating when criteria change (or forgetting to do so).

**Comment by Ainali** [comment]
I think this is because of the version they translated (we also used to do it manually). 
Meaning, when a newer translation is made, they should get it for "free".

**Comment by clausmullie** [comment]
Right, so let's not replace them retroactively so they remain as close to the version they were translated from? 

In that case, we can close this issue

**Comment by ixuz** [comment]
As mentioned by @Ainali and @clausmullie let's not retroactively change the old translated versions.
